tests/unittests/test_cli.py

```diff
@@ -224,7 +224,8 @@ def test_wb_devel_schema_subcommand_doc_content(self):
         self._call_main(['cloud-init', 'devel', 'schema', '--docs', 'all'])
         expected_doc_sections = [
             '**Supported distros:** all',
-            '**Supported distros:** almalinux, alpine, centos, debian, fedora',
+            ('**Supported distros:** almalinux, alpine, centos, debian, '
+             'fedora, rocky'),
```

Comment:
This is checking that the list in cc_ntp.py matches some documentation output. Unfrotunately it hasn't been updated in a while, but this should fix it.
```suggestion
            ('**Supported distros:** almalinux, alpine, centos, debian, '
             'fedora, opensuse, rhel, rocky, sles, ubuntu'),
```